addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2932,6 +2932,16 @@ registry.sizing = SnippetOptionWidget.extend({
             $body.on('mouseup', bodyMouseUp);
         });
 
+        // Disable the pointer events on the handle during the mouse wheel event.
+        // To prevent page scrolling from stopping when hovering over a handle.
+        this.$handles.on('wheel', _.throttle(() => {
+            this.$handles.css('pointer-events', 'none');
+            clearTimeout(this.wheelTimeout);
+            this.wheelTimeout = setTimeout(() => {
+                this.$handles.css('pointer-events', '');
+            }, 250);
+        }, 50));
```

Comment:
Mmh, a bit hacky but not a bad idea :+1: However, don't do that in the sizing snippet option by adding an event handler on all handles. It's not related to any option: do that at the source on the main overlay element, so in the `SnippetEditor` class.